server/routes/kubeconfig/kubeconfig_routes.py

```
@@ -0,0 +1,230 @@
+        if not ctx_name:
+            continue
+        contexts.append({
+            "context_name": ctx_name,
+            "cluster_name": ctx_info.get("cluster", ctx_name),
+            "namespace": ctx_info.get("namespace"),
+            "server_url": cluster_map.get(ctx_info.get("cluster")),
+        })
+    return contexts
+
+
+def _process_kubeconfig_entry(entry, user_id, org_id, registered, errors):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_process_kubeconfig_entry`

Add return type annotation: `None`

(ANN202)